From: Elias Probst <mail@eliasprobst.eu>
Date: Mon, 19 May 2014 18:24:01 +0000
Subject: Fix build (multiple missing Qt5:: components in target_link_libraries).
X-Git-Url: http://quickgit.kde.org/?p=plasma-workspace.git&a=commitdiff&h=7b5e47c1e7612817889ea063ce84471f5e007be9
---
Fix build (multiple missing Qt5:: components in target_link_libraries).

REVIEW: 118209
---


--- a/containmentactions/paste/CMakeLists.txt
+++ b/containmentactions/paste/CMakeLists.txt
@@ -5,10 +5,12 @@
 )
 
 add_library(plasma_containmentactions_paste MODULE ${paste_SRCS})
-target_link_libraries(plasma_containmentactions_paste 
-	KF5::Plasma 
-	KF5::KIOCore
-	)
+target_link_libraries(plasma_containmentactions_paste
+    Qt5::Gui
+    Qt5::Widgets
+    KF5::Plasma
+    KF5::KIOCore
+    )
 
 kservice_desktop_to_json(plasma_containmentactions_paste plasma-containmentactions-paste.desktop)
 

--- a/containmentactions/switchactivity/CMakeLists.txt
+++ b/containmentactions/switchactivity/CMakeLists.txt
@@ -9,6 +9,7 @@
 kservice_desktop_to_json(plasma_containmentactions_switchactivity plasma-containmentactions-switchactivity.desktop)
 
 target_link_libraries(plasma_containmentactions_switchactivity
+                      Qt5::Widgets
                       KF5::Plasma
                       KF5::KIOCore
                       KF5::Activities

--- a/dataengines/activities/CMakeLists.txt
+++ b/dataengines/activities/CMakeLists.txt
@@ -20,6 +20,7 @@
                       KF5::I18n
                       KF5::Service
                       Qt5::DBus
+                      Qt5::Widgets
                       )
 kservice_desktop_to_json(plasma_engine_activities plasma-engine-activities.desktop)
 

--- a/dataengines/favicons/CMakeLists.txt
+++ b/dataengines/favicons/CMakeLists.txt
@@ -7,6 +7,7 @@
 target_link_libraries(plasma_engine_favicons
     KF5::Plasma
     KF5::KIOCore
+    Qt5::Gui
 )
 
 kservice_desktop_to_json(plasma_engine_favicons plasma-dataengine-favicons.desktop)

--- a/dataengines/share/CMakeLists.txt
+++ b/dataengines/share/CMakeLists.txt
@@ -36,9 +36,11 @@
 
 add_library(plasma_packagestructure_share MODULE ${sharepackage_SRCS})
 target_link_libraries(plasma_packagestructure_share
-        KF5::Plasma
-        KF5::I18n
-        KF5::Service)
+                      KF5::Plasma
+                      KF5::I18n
+                      KF5::Service
+                      Qt5::Gui
+)
 
 install(TARGETS plasma_packagestructure_share
                  DESTINATION ${PLUGIN_INSTALL_DIR}/plasma/packagestructure)

--- a/runners/calculator/CMakeLists.txt
+++ b/runners/calculator/CMakeLists.txt
@@ -30,14 +30,15 @@
                           KF5::KIOCore
                           KF5::Runner
                           KF5::I18n
+                          Qt5::Network
                           Qt5::Script
-                          Qt5::Network
     )
 else ()
     add_library(krunner_calculatorrunner MODULE ${krunner_calculatorrunner_SRCS})
     target_link_libraries(krunner_calculatorrunner
                           KF5::Runner
                           KF5::I18n
+                          Qt5::Gui
                           Qt5::Script
     )
 endif ()

